Renamed the installing deploy folders and section to getting started

jonat123 opened the pull request.

sofietoft commented:
Looks good 💪 
Remember version 10 as well, and also redirects for the articles!

jonat123 commented:
@sofietoft I've moved it to the v10 section now as well and I have added redirects, could you please have a look again just to sanity check whether i have done everything correct? :D

sofietoft commented on 10/umbraco-deploy/.gitbook.yaml:
```diff
@@ -4,3 +4,11 @@ root: ./
 
+installing-deploy/installing-deploy: getting-started/install-configure-md
+installing-deploy/install-configure: getting-started-install-configure.md
+installing-deploy/install-configure/existing-site: getting-started/install-configure.md
+installing-deploy/install-configure/new-site: getting-started/install-configure.md
```
You might be able to add the direct links here, if we want want. So, link to the specific expandable, which should also automatically open the expandable, I think 🤔 

Example: `getting-started/install-configure#new-site`

sofietoft commented on 10/umbraco-deploy/.gitbook.yaml:
```diff
@@ -4,3 +4,11 @@ root: ./
+installing-deploy/install-configure: getting-started-install-configure.md
+installing-deploy/install-configure/existing-site: getting-started/install-configure.md
+installing-deploy/install-configure/new-site: getting-started/install-configure.md
+installing-deploy/cicd-pipeline: getting-started/README.md
```
Shouldn't this be `getting-started/cicd-pipeline/README.md` ? 🤔

sofietoft reviewed (commented):
Looks good 💪 
I've left a couple of comments - only on 10, but it should be applied for 11 as well.'

jonat123 commented:
merging in!
